lambdas/service/app.py

```diff
@@ -957,7 +959,59 @@ def generate_manifest(event, context):
     return {'Location': presigned_url}
 
 
-@app.route('/dss/files/{uuid}', methods=['GET'], cors=True)
+dss_files_spec = {
+    'tags': ['DSS'],
+    'parameters': [
+        params.path('uuid', str, description='UUID of the file to be checked out'),
+        params.query('fileName',
+                     schema.optional(str),
+                     description='The desired name of the file. If absent, the UUID of the file will be used.'),
```

Comment:
Don't duplicate code or documentation. Remove the docstring sections after porting them to the spec. Ideally, the entire docstring should be ported and removed alltogether.